src/main/java/com/powsybl/sld/server/entities/nad/NadConfigEntity.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.sld.server.entities.nad;
+
+import com.powsybl.sld.server.dto.nad.NadConfigInfos;
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.util.*;
+
+/**
+ * @author Charly Boutier <charly.boutier at rte-france.com>
+ */
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Entity
+@Builder
+@Table(name = "nadConfig")
+public class NadConfigEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id")
+    private UUID id;
+
+    @ElementCollection(fetch = FetchType.EAGER)
```

Comment:
```suggestion
```
I don't think we should ever use fetch EAGER

Reply:
Ok, updated.